app/models/user.rb

```diff
@@ -482,6 +485,42 @@ def title
     name
   end
 
+  def number_visits_per_day
+    return user.sign_in_count.round(1) if user.created_at.today?
+
+    (user.sign_in_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round(1)
```

Comment:
divides the number of times the user signed in by the number of days elapsed since the `user` was created in the `Users` table